### PR TITLE
chore(*): Upgrade Dependencies and Refactor

### PR DESCRIPTION
**Pull request by wodenx**
## Changes
- Add tokenSpec utilities to bodiless
- Move store, backend client and git operations to core from gatsby-theme-bodiless
- reorganize code in fclasses
- use npm workspaces instead of lerna for bootstrap
- Upgrade package dependencies

BREAKING CHANGE
- Renamed 'asToken' to 'flowHoc'
- renamed and repurposed token related types
- modified the type of any Design object to accept TokenSpecs and strings (will break any code which relies on their always being HOC's)

### CODE REVIEW

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

File: cypress/integration/smokeTest/cards_spec.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ describe('Card testing', function () {
     cy.xpath(imagePlaceholderXpath)
       .should('have.attr', 'src')
       .and('match', imageOrigPathRegex);
-    cy.isImageVisible(imagePlaceholderXpath);
+    // cy.isImageVisible(imagePlaceholderXpath);
```
Comment (TatsianaMiraniuk) [inline cypress/integration/smokeTest/cards_spec.js:121]: @Guilherme-Almeida-Zeni , why that checks are excluded? They are required to make sure that an image is uploaded/available.

**Review by TatsianaMiraniuk — CHANGES_REQUESTED** [review]

**Review by TatsianaMiraniuk — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-lint-simple.yml [hunk 2]
```diff
@@ -57,6 +60,7 @@ jobs:
       - name: Test
         run: npm run test --quiet --colors
 
+      # Temp using monorepo since standalone is breaking cypress tests.
       - name: Cypress tests
-        run: |
-          npm run test:ci
+        run: npm run test:monorepo
```
[the remark this answers is not in the document]
Comment (Guilherme-Almeida-Zeni) [reply .github/workflows/build-test-lint-simple.yml:65]: @TatsianaMiraniuk , 

As discussed on standup, we will not invest more time to fix the setup issues on standalone site.

File: cypress/support/commands.js [hunk 3]
```diff
@@ -17,7 +17,7 @@
 // create various custom commands and overwrite
 // existing commands.
 //
-// For more comprehensive examples of custom
+// For more comprehensive sites of custom
```
[the remark this answers is not in the document]
Comment (Guilherme-Almeida-Zeni) [reply cypress/support/commands.js]: @TatsianaMiraniuk ,

Reverted that. Please, check it again. Thanks!

File: packages/bodiless-backend/__tests__/fixtures/git/get-conflicts/hooks/prepare-commit-msg.sample [hunk 4]
```diff
@@ -9,7 +9,7 @@
 #
 # To enable this hook, rename this file to "prepare-commit-msg".
 
-# This hook includes three examples. The first one removes the
+# This hook includes three sites. The first one removes the
```
[the remark this answers is not in the document]
Comment (Guilherme-Almeida-Zeni) [reply packages/bodiless-backend/__tests__/fixtures/git/get-conflicts/hooks/prepare-commit-msg.sample]: @TatsianaMiraniuk ,

Reverted that. Please, check it again. Thanks!

File: packages/bodiless-core-ui/src/LocalContextMenu.tsx [hunk 5]
```diff
@@ -43,7 +44,7 @@ const groupClasses = 'bl-px-3';
 // eslint-disable-next-line max-len
 // const groupClasses = 'bl-border-t first:bl-border-t-0 bl-border-white bl-mt-grid-2 first:bl-mt-grid-0';
 
-// For accessibility attributes, see https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html
+// For accessibility attributes, see https://www.w3.org/TR/wai-aria-practices/sites/toolbar/toolbar.html
```
[the remark this answers is not in the document]
Comment (Guilherme-Almeida-Zeni) [reply packages/bodiless-core-ui/src/LocalContextMenu.tsx]: @TatsianaMiraniuk ,

Reverted that. Please, check it again. Thanks!

**Review by TatsianaMiraniuk — APPROVED** [review]

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

**Pull request merged**